client/js/components/procedure/AdministrationImport/AdministrationImport.vue

```diff
@@ -184,9 +184,12 @@ export default {
   },
 
   mounted () {
+    const promises = [this.loadComponents('email.import')]
+    if (hasPermission('feature_import_statement_pdf')) {
+      promises.push(this.loadComponents('import.tabs'))
+    }
     Promise.allSettled([
-      this.loadComponents('import.tabs'),
-      this.loadComponents('email.import')
+      promises
     ])
```

Comment:
shouldn't the [] brackets be removed here? `promises` are already an array if I am not mistaken